docs/admin/pool_operations.md

```diff
@@ -163,6 +163,9 @@ set aside depends on the size of the target and may take up 2+ GB.
 Therefore, Out of space conditions may occur even while pool query may not
 show min approaching zero.
 
+!!! caution
+    When space reserving is disabled for tiny pools, it may not function if the pool is filled up, since destroy/punch also requires additional space in a versioned storage.
```

Comment:
I don't think that it is a "when", but more like an assertion for tiny pools. We should also give an idea of what "tiny" means here.